dev-gap/radiroot/radiroot-2.9.ebuild

```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gap-pkg
+
+DESCRIPTION="Roots of a polynomial as radicals in GAP"
+SLOT="0"
+SRC_URI="https://github.com/gap-packages/${PN}/releases/download/v${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2"
```

Comment:
should be `GPL-2+` too, see https://github.com/gap-packages/radiroot/blob/fbbe58336a138d9c2a56eb41c851dd5be560fb2a/PackageInfo.g#L14

Reply:
Yep, fixed